client/apps/city_issues/forms/forms.py

```diff
@@ -0,0 +1,39 @@
+from django import forms
+
+from city_issues.models.issues import Issues, Category
+
+
+class IssueForm(forms.ModelForm):
+    class Meta:
+        model = Issues
+        fields = ['description', 'category', 'location_lat', 'location_lon', 'name']
+
+    name = forms.CharField(
+        max_length=350,
```

Comment:
IMHO so long name not needed, may be change to max_length=50

Reply:
I agree, maybe typos